docs/concepts/redirects.mdx

```diff
@@ -40,105 +38,125 @@ will fail, however, for https://anotherapp.com will succeed on any path.
 Ory Cloud has a total of six flows, **Login**, **Registration**,
 **Verification**, **Recovery**, **Settings** and **Logout** which can be
 configured to redirect back to any URL. A common use case would be to redirect
-the user to your application home screen after a logout or to a specific url on
-a sub-domain after a settings password update. On project creation the redirect
-flows are set to the default Ory Cloud Managed UI. For more information on the
-Ory Managed UI, please see the [Managed User Interface](./managed-ui.mdx)
-documentation.
-
-### Login, Registration and Settings
-
-The Login and Registration flows have three fields (all optional):
-
-1. `Default Redirect URL` domain or path relative to your application URL
-2. An `Post-Password Redirect URL`
-3. An `Post-OIDC Redirect URL` (**not supported**)
-
-And Settings has three fields (all optional):
+the user to your application home screen after a logout or to a specific URL on
+a sub-domain after a settings password update.
 
-1. `Default Redirect URL` domain or path relative to your application URL
-2. An `Post-Password Redirect URL`
-3. An `Post-Profile Redirect URL`
+On project creation the redirect flows are set to the default
+[Managed User Interface](./managed-ui.mdx).
 
-:::info
+### Login, Registration, and Settings
 
-Setting `Post-OIDC` redirects for Login and Registration will be supported in a
-future release. For more information see the
-[Ory Cloud Roadmap](../cloud/cloud-roadmap.mdx).
-
-:::
+The Login and Registration flows have three fields (all optional):
 
-When setting the `Default Redirect URL` as an FQDN (fully qualified domain name)
-such as `https://domain.example/login` it will overwrite the
-`Global Redirect URL`. As a path it will replace the `Global Redirect URL` path,
-for example a path such as `/login` will alter
-`https://domain.example/global/path` to `https://domain.example/login`.
+1. Default Redirect URL domain or path relative to your application URL
+2. Post-Password Redirect URL
+3. Post-OIDC Redirect URL
+
+The Settings flow has three fields (all optional):
+
+1. Default Redirect URL domain or path relative to your application URL
+2. Post-Password Redirect URL
+3. Post-Profile Redirect URL
+
+To overwrite the Global Redirect URL set the Default Redirect URL to a fully
```

Comment:
I find this a bit confusing, what's the difference between "Global Redirect URL" and "Default Redirect URL"?

Reply:
See here : https://github.com/ory/docs/pull/668/files#r830157781

 I see to clarify this a bit and see how it works without the Global.

Reply:
Oh I see, so the key in Ory Kratos is always `default_browser_return_url`, and then we have different prefixes for it. I think this is a bit confusing when reading it in the docs. It might be a good idea to present this in a diagram or something - even a YAML example would help.

Maybe this document helps also?

https://www.ory.sh/docs/kratos/concepts/browser-redirect-flow-completion